<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="textInput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>761</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>TH Sarabun New</family>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_tag1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Tag1</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="output_tag1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>361</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="output_tag2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>230</y>
      <width>361</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_tag2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>190</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Tag2</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
